linode/manager
# upcoming: [M3-7423] - Create Bucket - Able to choose a single Compute Region ID eg ('us-east')

## Description 📝
OBJ Multi-Cluster - Create Bucket  flow - Able to choose a single Compute Region ID eg ('us-east') instead of cluster ID. 

| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/119517080/39d1f7ef-1f6e-43ad-84b6-0c60f518d9dc)|![image](https://github.com/linode/manager/assets/119517080/7b789239-70ba-4475-93df-b527dd483332)|

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Turn on the OBJ Multi Cluster feature flag
- Enable MSW
- Navigate to http://localhost:3000/object-storage/buckets/create.

### Verification steps 
(How to verify changes)
- Verify there is no regression in existing functionality (Create Bucket flow with feature flag disabled and disable MSW).
- Verify the new GUI as per the mockup.
- Verify Create Bucket payload and response wrt to API spec.

## As an Author I have considered 🤔

*Check all that apply*
